CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ and this project adheres to
   performance. Also, remove `memory_limit` from `InstanceOptions`, and define it
   instead at `Cache` level (same memory limit for all cached instances).
   ([#697])
+- cosmwasm-vm: Bump required marker export `cosmwasm_vm_version_4` to
```

Comment:
Since this is not the final "version_5", why not `interface_version_5-alpha1` or such. Then if we have two incompatible pre-releases of 0.14 (likely) we can differentiate

Reply:
Hmm, sounds like a bit of an overhead to me. Open to do that is we see a clear need for it. This marker export is only a user support thing really. And no user outside of our team should ever have contact to a non-final standard library or VM. All intermediate incompatibilities are detected anyways at runtime.